spec/advanced_search_spec.rb

```diff
@@ -131,15 +131,15 @@
       end
       it 'subject as a phrase' do
         expect(@sub_phrase.size).to be >= 500
```

Comment:
Consider pumping up the minimum also - maybe 550?

Reply:
ok.